### PR TITLE
Add augeas

### LINKED ISSUE #26: Puppet 4 and Augeas
Hello:

I'm working on https://github.com/puppet-community/puppet-rundeck/pull/134 and the module was recently modified to use augeas. Augeas isn't included by default in puppet 4, so I added the gems to the gemfile. Travis is unable to build them because libaugeas isn't installed.

Is there a better way to do this than enabling sudo and installing libaugeas?

### PR DESCRIPTION
**Pull request by jyaworski**
Fixes #26.

### CODE REVIEW

**Comment by igalic** [comment]
@raphink is our request upstream (at Travis) still not merged?
@jyaworski when it does, we won't need sudo

**Comment by jyaworski** [comment]
@igalic how could we tell if the request is merged? Should I just try removing the sudo bits?

**Comment by igalic** [comment]
https://github.com/travis-ci/apt-package-whitelist

**Comment by igalic** [comment]
dunno, can't find it rn, i'm on a phone, only

**Comment by igalic** [comment]
yes! it's merged, we won't need sudo! yay

https://github.com/travis-ci/apt-package-whitelist/pull/343

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
 ---
 notifications:
   email: false
-sudo: false
+sudo: true
```
Comment (igalic) [inline moduleroot/.travis.yml]: back to false!

File: moduleroot/.travis.yml [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 language: ruby
 cache: bundler
 bundler_args: --without system_tests
-before_install: rm Gemfile.lock || true
```
Comment (igalic) [inline moduleroot/.travis.yml]: leave as is

File: moduleroot/.travis.yml [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install libaugeas-dev
+  - rm Gemfile.lock || true
 script:
```
Comment (igalic) [inline moduleroot/.travis.yml]: use http://docs.travis-ci.com/user/migrating-from-legacy/#Adding-APT-Packages instead

**Comment by jyaworski** [comment]
Thanks. I missed the new docs, I suppose. I've refactored it. It works in puppet-rundeck.

**Comment by raphink** [comment]
Totally :+1:  for this of course!

**Comment by raphink** [comment]
Now if https://github.com/travis-ci/apt-source-whitelist/issues/101 could be merged too, we would be able to test various versions of Augeas even.

**Comment by jyaworski** [comment]
Hello:

What's the status of this PR?

**Comment by igalic** [comment]
@jyaworski it's fixed!!!!
we can now use the travis way!

File: moduleroot/.travis.yml [hunk 4]
```diff
@@ -5,6 +5,10 @@ sudo: false
 language: ruby
 cache: bundler
 bundler_args: --without system_tests
+addons:
+  apt:
```
Comment (igalic) [inline moduleroot/.travis.yml]: bleargh.
we're still missing:
```yaml
     - sources:
       - augeas
```

Comment (raphink) [inline moduleroot/.travis.yml]: Yep. If people could comment on https://github.com/travis-ci/apt-source-whitelist/issues/101 to try and get there...

**Pull request merged**